api/apps/users/test/model.coffee

```diff
@@ -48,6 +48,14 @@ describe 'User', ->
         user.twitter_uid.should.equal '321'
         done()
 
+    it 'does not get user social media UIDs', (done) ->
+      user = _.extend fixtures().users, type: 'Admin'
```

Comment:
I was wrong about having to change antigravity. Because `findOrInsert` looks up users in Positron's db, we can just insert a stub user. (The other approach could work too, but this is simpler).